public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="padding-bottom: 50px">
```

Comment:
Please add styling to the 'App.css' not directly in html
it's not very good practice for react apps